src/components/header/style.tsx

```diff
@@ -0,0 +1,24 @@
+import styled from 'styled-components/native';
+export const ImageStyle = styled.Image`
+  margin-top: 20px;
+  width: 48px;
+  height: 48px;
+  border-radius: 100px;
+`;
+
+export const Head = styled.View`
+  padding: 26px;
+  height: 125px;
```

Comment:
It would be more convenient for the height to be dynamic with the components, you could use padding to make it fit well.